Makefile

```diff
@@ -57,15 +57,15 @@ test: generate fmt vet manifests ## Run unit and integration tests
 
 all: manager ## Default make target if no options specified
 
-test-validation: generate fmt vet manifests  ## Run validation tests
+test-validation: generate fmt vet manifests ginkgo  ## Run validation tests
```

Comment:
nit on the commit subject: `Update the makefile` is a bit generic. I'd change it to something like: "Makefile: use ginkgo instead of go test" and in the body explain why you require the change

Reply:
Changed back to go test.